python/mxnet/image.py

```diff
@@ -89,7 +89,31 @@ def fixed_crop(src, x0, y0, w, h, size=None, interp=2):
 
 
 def random_crop(src, size, interp=2):
-    """Randomly crop src with size. Upsample result if src is smaller than size."""
+    """Randomly crop src with size. Upsample result if src is smaller than size.
+
+    Parameters
+    ----------
+    src : source ndarray
+    size : (width, height) - size of the crop. If it is larger than the image, then 
+           the image is upsampled to size and returned
+    interp: interpolation to be used in case the size is larger (default: bicubic). Uses 
+           OPENCV convention for the parameters. NEAREST - 0, Bilinear - 1, bicubic - 2, area - 3
```

Comment:
The interpolation options should all have similar casing:
Nearest - 0, Bilinear - 1, Bicubic - 2, Area - 3 (preferred) or
nearest - 0, bilinear - 1, bicubic - 2, area - 3.